File: Codes/Ruby/3.rb
#!/usr/bin/env ruby

def fizz_buzz(max)
  arr = []
  (1..max).each do |n|
    text = ""
    if (n % 3 == 0)
      text << "Fizz"
    end
    if (n % 5 == 0)
      text << "Buzz"
    end
    if !((n % 3 == 0) || (n % 5 == 0))
      text = n
    end
    arr << text
  end
  return arr
end

fizz_buzz(100)

File: Codes/C#/3.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.RegularExpressions;

namespace Rextester
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Hello, world!");
            for(int i=1; i<=100;i++)
            {
                if(i%3==0&&i%5==0)
                {
                    Console.WriteLine("FizzBuzz");
                }
                else if(i%3==0&&i%5!=0)
                {
                    Console.WriteLine("Fizz");
                }
                else if(i%3!=0&&i%5==0)
                {
                    Console.WriteLine("Buzz");
                }
                else
                {
                    Console.WriteLine(i);
                }
            }
        }
    }
}
